scripts/Diffraction/isis_powder/pearl_routines/pearl_output.py

```diff
@@ -50,49 +49,41 @@ def _focus_mode_mods(output_file_paths, calibrated_spectra):
     return output_list
 
 
-def _focus_mode_trans(output_file_paths, atten, instrument, calibrated_spectra):
-    summed_ws = mantid.CloneWorkspace(InputWorkspace=calibrated_spectra[0])
-    for i in range(1, 9):  # Add workspaces 2-9 to workspace 1
-        summed_ws = mantid.Plus(LHSWorkspace=summed_ws, RHSWorkspace=calibrated_spectra[i])
-
-    summed_ws = mantid.Scale(InputWorkspace=summed_ws, Factor=0.111111111111111)
-
-    if atten:
-        # Clone a workspace which is not attenuated
-        no_att = output_file_paths["output_name"] + "_noatten"
-        mantid.CloneWorkspace(InputWorkspace=summed_ws, OutputWorkspace=no_att)
+def _focus_mode_all(output_file_paths, processed_spectra, perform_attenuation, instrument):
+    summed_spectra_name = output_file_paths["output_name"] + "_mods1-9"
+    summed_spectra = mantid.MergeRuns(InputWorkspaces=processed_spectra[:9], OutputWorkspace=summed_spectra_name)
 
-        summed_ws = mantid.ConvertUnits(InputWorkspace=summed_ws, Target="dSpacing")
-        summed_ws = instrument._attenuate_workspace(summed_ws)
-        summed_ws = mantid.ConvertUnits(InputWorkspace=summed_ws, Target="TOF")
+    summed_spectra = mantid.Scale(InputWorkspace=summed_spectra, Factor=0.111111111111111,
```

Comment:
Yes these sum up workspaces 1-9 (inclusive) so the value in the original scripts was 1/9 in decimal 
The other one sums up 3 workspaces so 1/3 in decimal